server/util/query_builder/query_builder_test.go

```diff
@@ -16,7 +16,7 @@ func normalize(t *testing.T, query string) string {
 	// The current logic in this func blindly strips whitespace, which might
 	// change the meaning of queries containing strings. So for now, just fail
 	// the test if we see any string delimiters.
-	if strings.Contains(query, "\"") || strings.Contains(query, "'") || strings.Contains(query, "`") {
+	if strings.Contains(query, "'") || strings.Contains(query, "`") {
```

Comment:
`"` is now an identifier delimiter instead of a string delimiter and is thus safe so long as none of our identifiers contain whitespace.